package.json

```diff
@@ -106,7 +106,8 @@
     "redux-thunk": "^2.3.0",
     "stream-browserify": "^3.0.0",
     "uuid": "^8.3.1",
-    "weak-napi": "^2.0.2"
+    "weak-napi": "^2.0.2",
+    "cypress": "^9.0.0"
```

Comment:
Is there a reason why cypress is included here as a production dependency? It is all ready included later as an optional dependency https://github.com/LD4P/sinopia_editor/blob/66371b31d1552ba6a97e1ea53d52951acbe0501a/package.json#L214-L216

Reply:
I inadvertenly caused that addition with some npm process-- I will remove it.